tv_apps/fling-player/js/connector.js

```diff
@@ -155,6 +175,26 @@
           }
           this.fire('seekRequest', { time : time });
         break;
+
+        case 'device-info':
+          var resolve;
+
+          if (!this._controllingDeviceInfo) {
+            this._controllingDeviceInfo = {};
```

Comment:
Why don't you just initialize it with ```
this._controllingDeviceInfo = {
  displayName = msg.displayName
}
```?

And the `if` clause is just unnecessary.